demos/dialog/index.html

```diff
@@ -153,6 +155,29 @@
         Secondary
       </mwc-button>
     </mwc-dialog>
+
+    <mwc-button data-num="8" raised>Form Validation</mwc-button>
+    <mwc-dialog id="dialog8" heading="Form Validation">
+      <p>This dialog can validate user input before closing.</p>
+      <mwc-textfield
+        id="dialog8-text-field"
+        minlength="3"
+        maxlength="64"
+        placeholder="First Name"
```

Comment:
nit: prefer sentence-style capitalization (go/material/writing/grammar-punctuation): "First name"

Reply:
sweet! let's do it